src/bootnodoor/bootnodoor_launcher.star

```diff
@@ -0,0 +1,188 @@
+        genesis_validators_root,
+        el_genesis_hash,
+        external_bootnodes,
+        global_node_selectors,
+        tolerations,
+        docker_cache_params,
+    )
+
+    plan.add_service(SERVICE_NAME, config)
+
+    # Request ENR from the running bootnodoor service
+    # The /enr endpoint returns a plain string (not JSON)
+    enr_recipe = GetHttpRequestRecipe(
+        endpoint="/enr",
+        port_id=constants.HTTP_PORT_ID,
+    )
+
+    response = plan.request(
+        recipe=enr_recipe,
+        service_name=SERVICE_NAME,
+    )
+
+    bootnodoor_enr = response["body"]
+
+    # Request ENODE from the running bootnodoor service for EL bootnode
+    enode_recipe = GetHttpRequestRecipe(
+        endpoint="/enode",
+        port_id=constants.HTTP_PORT_ID,
+    )
+
+    enode_response = plan.request(
+        recipe=enode_recipe,
+        service_name=SERVICE_NAME,
+    )
+
+    bootnodoor_enode = enode_response["body"]
+
+    return bootnodoor_enr, bootnodoor_enode
+
+
+def get_config(
+    bootnodoor_params,
+    el_cl_genesis_data,
+    private_key,
+    genesis_validators_root,
+    el_genesis_hash,
+    external_bootnodes,
+    node_selectors,
+    tolerations,
+    docker_cache_params,
+):
+    cmd = [
+        "--cl-config",
+        "{0}/config.yaml".format(constants.GENESIS_DATA_MOUNTPOINT_ON_CLIENTS),
+        "--genesis-validators-root",
+        genesis_validators_root,
+        "--el-config",
+        "{0}/genesis.json".format(constants.GENESIS_DATA_MOUNTPOINT_ON_CLIENTS),
+        "--el-genesis-hash",
+        el_genesis_hash,
+        "--private-key",
+        private_key,
+        "--bind-addr",
+        "0.0.0.0",
+        "--web-ui",
+        "--nodedb",
+        "/nodes.db",
+    ]
+
+    # Add external bootnodes for non-kurtosis networks (ephemery, devnets)
+    if external_bootnodes != None:
+        cmd.append("--bootnodes")
+        cmd.append(external_bootnodes)
+
+    # Add any extra args from the params
+    if len(bootnodoor_params.extra_args) > 0:
+        cmd.extend(bootnodoor_params.extra_args)
+
+    return ServiceConfig(
+        image=shared_utils.docker_cache_image_calc(
+            docker_cache_params,
+            bootnodoor_params.image,
+        ),
+        ports=USED_PORTS,
+        files={
+            constants.GENESIS_DATA_MOUNTPOINT_ON_CLIENTS: el_cl_genesis_data.files_artifact_uuid,
+        },
+        cmd=cmd,
+        min_cpu=bootnodoor_params.min_cpu,
+        max_cpu=bootnodoor_params.max_cpu,
+        min_memory=bootnodoor_params.min_mem,
+        max_memory=bootnodoor_params.max_mem,
+        node_selectors=node_selectors,
+        tolerations=tolerations,
+    )
+
+
+def get_el_genesis_hash(plan, network_params, el_cl_genesis_data):
+    # For mainnet and sepolia, use static genesis hashes
+    if network_params.network == "mainnet":
+        return "0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3"
+    elif network_params.network == "sepolia":
```

Comment:
any reason we don't need static genesis hashes for hoodi?

Reply:
hoodi & all our devnets have a post-bellatrix genesis,  and therefore the genesis block hash is included in deposit_contract_block_hash.txt
for mainnet & sepolia we're not storing the genesis block hash in the repository